voicevox/http.py

```diff
@@ -24,8 +24,8 @@ async def close(self) -> None:
         await self.session.aclose()
 
     async def request(self, method: str, path: str, **kwargs) -> dict:
-        response = await self.session.request(method, path, **kwargs)
         logger.debug(f"Request: {method} Path: {path} kwargs: {kwargs}")
+        response = await self.session.request(method, path, **kwargs)
         logger.debug(f"Response: {response.content}")
```

Comment:
ステータスコードもあった方がいいんじゃない。
```suggestion
        logger.debug("Response: {0.status_code}: {0.content}".format(response))
```

Reply:
せやな